includes/Modules/Analytics_4.php

```diff
@@ -1161,11 +1161,20 @@ function ( $metric_def ) {
 
 						return $metric;
 					},
-					array_filter( $metrics )
+					$metrics
```

Comment:
As above, we are now validating the empty value case, so don't need to filter them out here.